uber/automated_emails.py

```diff
@@ -269,7 +269,7 @@ def __init__(self, subject, template, filter, ident, **kwargs):
         #     Group.status == c.APPROVED,
         #     Group.approved < (func.now() - timedelta(days=30)),
         #     Group.is_unpaid == True),
-        needs_approval=False,
```

Comment:
These should probably be reverted, or set back to False in the magprime plugin at least. I'll have to look into that before merging this.

Reply:
Did this in https://github.com/magfest/magprime/pull/286, and warned the MAGWest chair about the change!